model/registry/db.yaml

```diff
@@ -529,3 +533,67 @@ groups:
           reference the [elasticsearch schema](https://raw.githubusercontent.com/elastic/elasticsearch-specification/main/output/schema/schema.json)
           in order to map the path part values to their names.
         examples: ['db.elasticsearch.path_parts.index=test-index', 'db.elasticsearch.path_parts.doc_id=123']
+  - id: registry.db.vector
+    prefix: db.vector
+    type: attribute_group
+    brief: >
+      This group defines attributes for vector databases.
+    attributes:
+      - id: similarity
+        type:
+          allow_custom_values: true
+          members:
+            - id: cosine
+              value: 'cosine'
+              brief: >
+                The cosine metric.
+              stability: experimental
+            - id: dot
+              value: 'dot'
+              brief: >
+                The dot product metric.
+              stability: experimental
+            - id: euclidean
+              value: 'euclidean'
+              brief: >
+                The euclidean distance metric.
+              stability: experimental    
+            - id: manhattan
+              value: 'manhattan'
+              brief: >
+                The Manhattan distance metric.
+              stability: experimental
+        stability: experimental
+        brief: >
+          The metric used in similarity search.
+        examples: 'cosine'
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          The ID of vector.
+        examples: '5c56c793-69f3-4fbf-87e6-c4bf54c28c26'
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The name field as of the vector (e.g. a field name).
+        examples: 'vector'
+      - id: embeddings
+        type: double[]
```

Comment:
this would be huge and not usable on telemetry - is it necessary?

Reply:
If you refer to the fact that the embedding values can be huge (eg. 1536) I agree, but I think we need to have a way to represent a vector in the semantic convention. We can mark this as optional.

Reply:
why do we need to record it on the telemetry? It can be done with logs (and common ways to control verbosity), not as a span attribute

Reply:
+1, it could be recorded with events or logs. Curious to hear what use case drives the requirement to record this?

Reply:
There's not a specific use case, as semantic convention I would expect to have the embedding values represented in a proposal about vector database (at lest as optional) . Anyway, if you think that it's better to have it in logs for me it's fine.